src/components/Calculator.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../App.css';
+
+const Calculator = () => (
+  <div>
+    <div className="result">0</div>
+    <div className="row">
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button" className="operation">÷</button>
+    </div>
+    <div className="row">
+      <button type="button">7</button>
+      <button type="button">8</button>
```

Comment:
- [ ] According to the project's requirements, please add at least one child component in the `Calculator` component. You could create a re-usable `<Button />` component
![image](https://github.com/tajemouti/math-magicians/assets/46839548/11dd7354-76e5-44f7-b1f1-163f29bec44f)